.travis.yml

```diff
@@ -2,25 +2,31 @@
 #   Generic MoveIt Travis Continuous Integration Configuration File
 #   Works with all MoveIt! repositories/branches
 #   Author: Dave Coleman, Jonathan Bohren
+dist: trusty
 language:
   - cpp
   - python
 python:
   - "2.7"
 compiler:
   - gcc
+cache:
+  - ccache
 env:
   global:
     - ROSWS=wstool
     - BUILDER=catkin
+    - USE_DOCKER=true
+    - USE_TRAVIS=true
     - CATKIN_PARALLEL_JOBS='-p2'
     - CATKIN_PARALLEL_TEST_JOBS='-p1'
     - ROS_PARALLEL_JOBS="-j2"
     - ROS_PARALLEL_TEST_JOBS='-j1'
   matrix:
     - ROS_DISTRO=indigo  USE_DEB=true
     - ROS_DISTRO=indigo  USE_DEB=false NOT_TEST_INSTALL=true
-    - ROS_DISTRO=kinetic USE_DEB=false
+    - ROS_DISTRO=kinetic USE_DEB=false NOT_TEST_INSTALL=true
+    - ROS_DISTRO=melodic USE_DEB=false NOT_TEST_INSTALL=true BEFORE_SCRIPT="pwd; sed -i \"35iadd_definitions(-Wno-deprecated)\" hrpsys/CMakeLists.txt; (cd hrpsys; git diff)"  ROSDEP_ADDITIONAL_OPTIONS="-n -q -r --ignore-src --skip-keys libpng12-dev --skip-keys leap_motion" #BEFORE_SCRIPT: to build hrpsys on melodic. ROSDEP_ADDITIONAL_OPTIONS: leap_motion is not released on melodic.
```

Comment:
Where do you use `libpng12-dev`? I think that is the dependency of https://github.com/fkanehiro/openhrp3.
And please comment `leap_motion` is run_depend of `hironx_tutorial`:
https://github.com/start-jsk/rtmros_tutorials/blob/7dc73f471347af969570b5a675fcafe092434469/hironx_tutorial/package.xml#L26

Reply:
Thank you. I fixed https://github.com/start-jsk/rtmros_tutorials/pull/571/commits/abd1870bfd0ae2924554e66bf46c064ee6e90522